### DISCUSSION TITLE
[Feature Request] Parse .osu Files As Taitai (multiple files)

### DISCUSSION THREAD

**Discussion by TRRESTGHGYTCFTGBCXSZD** [TeamRizu/OutFox#405] [03 - Feature Requests]
**Is your feature request related to a problem? Please describe.**

There's Taitai mode here but it has no osu parser. please add this

**Describe the solution you'd like**

Parse BPM Inside [TimingPoints] (60000/BPM)
Because osu Resets Measure To 0 When Bpm Changes, It Needs To Rounded By 192th To Change Bpm.
IGNORE NEGATIVE VALUES, THOSE ARE NOT BPM.
How To Do?
[TimingPoints]
6966,166.527893422148,8,1,0,100,1,0
18956,-100,4,1,0,100,0,1
↓ ↓ ↓
#BPMS:6.966=360.300,

Always Parse As Edit With Description Containing [Metadata] Version:
Like
[Metadata]
Version:Ono's Taiko Oni;
↓ ↓ ↓
#DESCRIPTION:Ono's Taiko Oni;

**Additional context**

[41823 The Quick Brown Fox - The Big Black.zip](https://github.com/TeamRizu/OutFox/files/6951905/41823.The.Quick.Brown.Fox.-.The.Big.Black.zip)

**Comment by Scraticus** [comment]
Hmm, this is something we'll definitely look into in the future for sure. I'm going to be working on the TJA parser first, then this should be definitely higher profile.

Thanks for the suggestion

**Comment by Scraticus** [comment]
hiya again!

just wondering where you got the information on the 'timing points' side in your example above? 

I'm currently working on the parser and not finding much info on it

**Reply by TRRESTGHGYTCFTGBCXSZD** [reply]
https://osu.ppy.sh/wiki/en/Client/File_formats/Osu_%28file_format%29#osu!taiko
for reference
